appveyor.yml

```diff
@@ -34,6 +34,7 @@ build_script:
   - IF EXIST .git\shallow (git fetch --unshallow)
   - C:\projects\julia\bin\julia -e "versioninfo();
       Pkg.clone(pwd(), \"SQLite\"); Pkg.build(\"SQLite\")"
+  - where *sqlite3*
```

Comment:
very strange. I wonder if the appveyor process somehow has a copy from git in memory or something